Testing/PerformanceTests/analysis_mpl.py

```diff
@@ -529,9 +494,10 @@ def generate_html_subproject_report(path, last_num, x_field='revision', starts_w
     html += """<h2>Latest Results Summary</h2>"""
     html += get_html_summary_table(test_names)
 
-
     # -------- Report for each test ------------------------
     for name in test_names:
+        if name != 'AlgorithmsTest.CreateWorkspaceTestPerformance.testBigWorkspace':
```

Comment:
This seems to skip the for loop for all but one test. Just wanted to double check whether this is the intended behavior?

Reply:
Nice catch! I had this in for debugging so it didn't run over every test as it took ages.